AppFog/understanding-appfog-quotas.md

```diff
@@ -0,0 +1,43 @@
+{{{
+  "title": "Understanding AppFog Quotas",
+  "date": "09-08-2015",
+  "author": "Robert Brumfield",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false
+}}}
+
+### Audience
+
+Application developers
+Control administrators
+
+### Quotas
+
+Every Account that has enabled an Account gets an Organization in AppFog.  Organizations in AppFog contain Spaces that serve as a way to organize applications.  By default, each Organization gets three Spaces, Dev, QA, and Prod.  A user may add additional Spaces.  Quotas may be applied at both the Organization level and the Space level.
+
+Every Organization has a quota that defines the resources they can use when deploying apps.  This is defined by AppFog and cannot be changed except via ticket to https://support.ctl.io. The resources limited by quota are total memory, instance memory, routes, and service instances.  
```

Comment:
https://support.ctl.io may not always get auto-linked. can you change it to `[https://support.ctl.io](https://support.ctl.io)`